production/install

```diff
@@ -1198,7 +1209,61 @@ if [ "${BITCOIN_MAINNET_ENABLE}" = ON ];then
         ;;
 
         Debian)
-            osSudo "${ROOT_USER}" install -c -o "${ROOT_USER}" -g "${ROOT_GROUP}" -m 644 "${MEMPOOL_HOME}/${MEMPOOL_REPO_NAME}/production/bitcoin-mainnet.service" "${DEBIAN_SERVICE_HOME}"
+            osSudo "${ROOT_USER}" install -c -o "${ROOT_USER}" -g "${ROOT_GROUP}" -m 644 "${MEMPOOL_HOME}/${MEMPOOL_REPO_NAME}/production/linux/bitcoin.service" "${DEBIAN_SERVICE_HOME}"
+        ;;
+    esac
+fi
+
+################################
+# Bitcoin instance for Testnet #
+################################
+
+if [ "${BITCOIN_MAINNET_ENABLE}" = ON ];then
+    echo "[*] Installing Bitcoin Testnet service"
+    case $OS in
+
+        FreeBSD)
+            echo "[*] FIXME: Bitcoin Testnet service must be installed manually on FreeBSD"
```

Comment:
```suggestion
```
on freebsd we start bitcoind from cron on reboot so this service script isn't used